src/background/adblocker.js

```diff
@@ -140,17 +142,13 @@ export const setup = asyncSetup([
           (enabledEngines.length !== prevEnabledEngines.length ||
             enabledEngines.some((id, i) => id !== prevEnabledEngines[i])))
       ) {
-        // The regional filters engine is no longer used, so we must remove it
```

Comment:
This code has been here since `10.4.5`, and we have not removed it yet because of the Firefox delay. As the `10.4.10` is this code will be on production in Firefox after at least 2-3 weeks - it is the best time to clear it up.